test/built-ins/Array/from/Array.from_arity.js

```diff
@@ -7,6 +7,19 @@ description: >
     The Array.from() method creates a new Array instance
     from an array-like or iterable object.
     https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from
+info: >
```

Comment:
I know you didn't author the description, but since you are changing the frontmatter, would you mind also shortening this description to one line, [as recommended in the project's contribution guidelines](https://github.com/tc39/test262/blob/d62f3a82b97dba22a0be2798fbe34548e96a439f/CONTRIBUTING.md#description)?